tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -331,17 +331,17 @@ def bluetooth_disable(self):
 
     @property
     def bluetooth_is_enabled(self):
-        return self.marionette.execute_script("return window.navigator.mozBluetooth.enabled")
+        return self.marionette.execute_script("return window.navigator.mozBluetooth.getAdapters()[0].state === 'enabled'")
```

Comment:
Maybe we could wrap `window.navigator.mozBluetooth.getAdapters()[0]` in an atom called `getFirstAdapter` and use it in the next 3 following methods. This would avoid the duplication